### PR TITLE
Format update & store last known alert in a txt file

### PR DESCRIPTION
**Pull request by MichaelVasile**

### CODE REVIEW

**Review by MichaelVasile — COMMENTED** [review]
Changes made have been tested locally and are known to be working.

File: ritcovid.py [hunk 1]
```diff
@@ -96,47 +96,37 @@ def get_statistics():
     return all_students, all_staff, new_students, new_staff, campus_quarantine, offcampus_quarantine, campus_isolated, \
            offcampus_isolated, isolation_beds, last_updated, new_case_stat, tests_administered
 
-def check_last_known():
-    # to anyone looking at this, this is probably not the best way to do this
-    # but it's temporary so shhh.
 
+def check_last_known():
+    # Check to see if the file does not exist or is empty.
     if os.path.exists("last_known.txt") == False or os.path.getsize("last_known.txt") == 0:
```
Comment (ss9415) [inline ritcovid.py:102]: This conditional is not needed. See comment on line 106.

File: ritcovid.py [hunk 2]
```diff
@@ -96,47 +96,37 @@ def get_statistics():
...
         print("last_known.txt does not exist or is empty. Will create it now...")
-        f = open("last_known.txt", "w+")
 
-        # If the file is blank, populate it
-        f.write(str(get_alert_level()[0]))
-        # Read in the data
-        last_known = f.readline()
-        # Close the file
-        f.close()
+        with open("last_known.txt", "w+") as f:
```
Comment (ss9415) [inline ritcovid.py:106]: `w+` creates the file if it does not exist

File: ritcovid.py (hunk 1 above)
Comment (MichaelVasile) [reply ritcovid.py:102]: The purpose of this conditional is to check if the file exists and isn't empty when initially loading the bot. We don't want it overriding the what is there if the file already exists.

**Review by ss9415 — APPROVED** [review]

**Pull request merged**